pages/windows/doskey.md

```diff
@@ -3,30 +3,30 @@
 > Manage macros, windows commands and command-lines.
 > More information: <https://docs.microsoft.com/windows-server/administration/windows-commands/doskey>.
 
-- List available macros:
+- Print all macros:
 
 `doskey /macros`
 
-- Create a new macro:
+- Create the new macro:
 
 `doskey {{name}} = "{{command}}"`
```

Comment:
```suggestion
`doskey {{name}} = {{command}}`
```
I tested on my windows10, the quotation marks aren't needed. Or it would be like this.
```
>doskey ls = "dir"
>ls
'"dir"' is not recognized as an internal or external command,
operable program or batch file.
```